### PR TITLE
Births to WLHIV

### PR DESCRIPTION
**Pull request by mwalte10**
Calculates the number of births to WLHIV, needed for MTCT. Please review the 15 year old entrants bit that has changed in the demographic model in particular to make sure that won't cause any issues. 

Included births in test child file to make sure it is never negative.

### CODE REVIEW

File: inst/include/hiv_demographic_projection.hpp
Comment (r-ash) [reply inst/include/hiv_demographic_projection.hpp:62]: Oh is this a bit of old debug code?

File: inst/include/hiv_demographic_projection.hpp
Comment (r-ash) [inline inst/include/hiv_demographic_projection.hpp:123]: Can this TODO be removed now?

**Review by r-ash — APPROVED** [review]
This looks great, couple of small suggestions but they are fairly inconsequential. I'm approving so not blocked on me whilst I am away next week but would be worth a review from science side before merging this.

File: inst/include/hiv_demographic_projection.hpp
Comment (mwalte10) [reply inst/include/hiv_demographic_projection.hpp:62]: good catch! removed

**Comment by mwalte10** [comment]
@jeffeaton  I've added you to look through the methods per Rob's comment of approval from the technical side. I'm incorporating his suggestions but let me know if you have any others.

**Pull request closed without merging**
